src/Resources/views/formatter/badge.html.twig

```diff
@@ -0,0 +1,20 @@
+{% set classes = '' %}
```

Comment:
Nit:

Other ways, possible more ergonomic ones:

- Creating an array, `|merge` classes into the array and use `|join(' ')` at the end to create the string.
- Use [`html_classes` function](https://twig.symfony.com/doc/3.x/functions/html_classes.html).